packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -753,6 +753,7 @@ import 'package:{{name}}/{{file}}';
 
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
+// ignore: public_member_api_docs
```

Comment:
nit: we could just move this to the top of the file and do an `// ignore_for_file: public_member_api_docs`.

We should test this by adding an analyzer integration test to something that uses this template too.

Reply:
I don't really want to commit to keeping this file analysis clean for any arbitrary project's analyzer rules. Ideally we would make it invisible to the user's project

Reply:
I think making the file invisible to the user's project would be fine, but I don't know how do this.

Reply:
This change is good as is, we'll look for a better long term solutions separately